rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -586,24 +673,42 @@ def _create_model_data(
                         self._tag_ids_for_crf(example, tag_spec)
                     )
 
-        X_sparse = np.array(X_sparse)
-        X_dense = np.array(X_dense)
-        Y_sparse = np.array(Y_sparse)
-        Y_dense = np.array(Y_dense)
+        X_sparse_sequence = np.array(X_sparse_sequence)
+        X_sparse_sentence = np.array(X_sparse_sentence)
+        X_dense_sequence = np.array(X_dense_sequence)
+        X_dense_sentence = np.array(X_dense_sentence)
+        Y_sparse_sequence = np.array(Y_sparse_sequence)
+        Y_sparse_sentence = np.array(Y_sparse_sentence)
+        Y_dense_sequence = np.array(Y_dense_sequence)
+        Y_dense_sentence = np.array(Y_dense_sentence)
         label_ids = np.array(label_ids)
         tag_name_to_tag_ids = {
             tag_name: np.array(tag_ids)
             for tag_name, tag_ids in tag_name_to_tag_ids.items()
         }
 
         model_data = RasaModelData(label_key=self.label_key)
-        model_data.add_features(TEXT_FEATURES, [X_sparse, X_dense])
-        model_data.add_features(LABEL_FEATURES, [Y_sparse, Y_dense])
+        model_data.add_features(
+            TEXT_SEQUENCE_FEATURES, [X_sparse_sequence, X_dense_sequence]
+        )
+        model_data.add_features(
+            TEXT_SENTENCE_FEATURES, [X_sparse_sentence, X_dense_sentence]
+        )
+        model_data.add_features(
+            LABEL_SEQUENCE_FEATURES, [Y_sparse_sequence, Y_dense_sequence]
+        )
+        model_data.add_features(
+            LABEL_SENTENCE_FEATURES, [Y_sparse_sentence, Y_dense_sentence]
+        )
 
-        if label_attribute and model_data.feature_not_exist(LABEL_FEATURES):
+        if (
+            label_attribute
+            and model_data.feature_not_exist(LABEL_SENTENCE_FEATURES)
+            and model_data.feature_not_exist(LABEL_SEQUENCE_FEATURES)
+        ):
             # no label features are present, get default features from _label_data
             model_data.add_features(
-                LABEL_FEATURES, self._use_default_label_features(label_ids)
+                LABEL_SEQUENCE_FEATURES, self._use_default_label_features(label_ids)
```

Comment:
shouldn't the default features be sentence features?
```suggestion
                LABEL_SENTENCE_FEATURES, self._use_default_label_features(label_ids)
```